### PR TITLE
docs: Added root internal dependencies explanation

### PR DESCRIPTION
**Pull request by NicholasLYang**
### Testing Instructions

Closes TURBO-1786

### CODE REVIEW

File: docs/pages/repo/docs/troubleshooting.mdx [hunk 1]
```diff
@@ -80,6 +80,18 @@ directions even if you aren't planning to make a contribution.
 Turborepo does not support nested workspaces. Therefore, if you have a workspace that contains another workspace,
 you may see unexpected behavior. We recommend flattening your workspace structure to avoid this issue.
 
+### Root Dependencies
+
+If you have a package that is used by several other packages in the monorepo, say a standard ESLint configuration,
```
Comment (tknickman) [inline docs/pages/repo/docs/troubleshooting.mdx:85]:
```suggestion
If you have an internal package that is used by several other packages in the monorepo, say a standard ESLint configuration,
```

Comment (NicholasLYang) [reply docs/pages/repo/docs/troubleshooting.mdx:85]: I might try to phrase it without "internal" because that term isn't really explained until later on

Comment (tknickman) [reply docs/pages/repo/docs/troubleshooting.mdx:85]: Agreed.

**Review by arlyon — APPROVED** [review]

**Comment by arlyon** [comment]
Another candidate for a `turbo optimize` command is detecting local packages in your root package json.

**Pull request merged**